cypress/integration/providers_spec.js

```diff
@@ -137,11 +138,12 @@ describe('Dashboard Providers Page', () => {
         .type(host);
 
       cy.get('form div button').contains('Submit').click();
```

Comment:
the `submit` doesn't take you to overview now.  you have to click submit, then success.  And I haven't had a chance to see why.

Reply:
Does this cause the user any problem?

Reply:
Couldn't resist taking a peek at this.

Looks like the issue is that in `componentDidUpdate` of `EditRecord`, it is grabbing `state` from `prevProps` instead of `this.props`. Now that we are rendering far fewer times, the prevProps correctly does not have a success status. Pulling the state out of this.props seems to fix it.

Reply:
Sweet. Nice find. @flamingbear do you care if @dopeters pushes this tweak?

Reply:
Sorry I get my emails slowly.  I chatted and @dopeters is pushing to a branch for me.

Reply:
probably overkill

Reply:
Added fix here https://github.com/nasa/cumulus-dashboard/pull/704